src/seedu/addressbook/data/person/Email.java

```diff
@@ -33,8 +36,13 @@ public Email(String email, boolean isPrivate) throws IllegalValueException {
     /**
      * Returns true if the given string is a valid person email.
      */
-    public static boolean isValidEmail(String test) {
-        return test.matches(EMAIL_VALIDATION_REGEX);
+    public static boolean isValidEmail(String test) { return test.matches(EMAIL_VALIDATION_REGEX); }
```

Comment:
This change is not related to this PR? Anyway the signature and body of a function should not be on the same line